tests/functional/api/test_groups.py

```diff
@@ -119,18 +120,18 @@ def test_it_returns_404_if_authority_mismatch_on_group(self, app, factories, use
 
         assert res.status_code == 404
 
-    def test_it_returns_403_if_missing_auth(self, app, user, group):
+    def test_it_returns_404_if_missing_auth(self, app, user, group):
         res = app.post_json("/api/groups/{pubid}/members/{userid}".format(pubid=group.pubid, userid=user.userid),
                             expect_errors=True)
 
-        assert res.status_code == 403
+        assert res.status_code == 404
```

Comment:
This is actually appropriate in this case, as a `404` here protects us from leaking information about whether a given ID'd group exists or not...